internal/services/network/express_route_gateway_resource.go

```diff
@@ -168,6 +175,7 @@ func resourceExpressRouteGatewayRead(d *pluginsdk.ResourceData, meta interface{}
 			virtualHubId = *props.VirtualHub.ID
 		}
 		d.Set("virtual_hub_id", virtualHubId)
+		d.Set("allow_none_virtual_wan_traffic", props.AllowNonVirtualWanTraffic)
```

Comment:
this is a typo?
```suggestion
		d.Set("allow_non_virtual_wan_traffic", props.AllowNonVirtualWanTraffic)
```
and the property name should be `allow_non_virtual_wan_traffic`